### ISSUE TITLE
The BSL

### ISSUE THREAD

**Issue by jayfk** [pybee/paying-the-piper#76]
Might be interesting to watch how this one goes:
- Blogpost: http://monty-says.blogspot.de/2016/08/applying-business-source-licensing-bsl.html
- Techcrunch article: https://techcrunch.com/2016/08/19/mysql-founder-tries-a-new-software-licensing-model/
- BSL FAQ: https://mariadb.com/bsl-faq-adopting

**Comment by nayafia** [comment]
Yes, very! We're planning on doing an episode on licenses on the [RFC podcast](https://changelog.com/rfc/) soon, will add this to convo topics.
